src/EFCore.PG.Trgm/EFCore.PG.Trgm.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
```

Comment:
There is no need in this since there is a `Directory.Build.Props` in the parent directory.

Reply:
593c77fc6768cf1f32e4a0c04a7fdf6ff0806aed fixes this too.